lib/nexthop_group.c

```diff
@@ -322,13 +339,7 @@ static void nexthop_group_unsave_nhop(struct nexthop_group_cmd *nhgc,
 		return;
 
 	list_delete_node(nhgc->nhg_list, node);
-
-	if (nh->nhvrf_name)
-		XFREE(MTYPE_TMP, nh->nhvrf_name);
-	if (nh->intf)
-		XFREE(MTYPE_TMP, nh->intf);
-
-	XFREE(MTYPE_TMP, nh);
+	nhgl_delete(nh);
```

Comment:
Shouldn't this change be it's own commit?  The consolidation of all the deletion into one function call?

Reply:
Such a trivial change... but ok.